src/htcondor_es/convert_to_json.py

```diff
@@ -575,7 +576,13 @@
 # Initialize aff_mgr
 aff_mgr = None
 try:
-    aff_mgr = AffiliationManager(recreate=False)
+    aff_mgr = AffiliationManager(
+        recreate_older_days=10,
```

Comment:
Does it mean that this class will re-create a data? In k8s we should avoid this since the cron will run outside of spider script and produce data independently. The spider script should only READ the data from certain location.